composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
This is the test server that I created during the last seven days (for code coverage in THIS project). It could stand at least one request per second, but nothing more. I don\'t know why some connections just ignored. I refactored the architecture many times, but I can not do nothing more than that.
I'm very much afraid that the tests in the following PR will fall, because my server will not be able to process them, and not because the code does not work.